.gitignore

```diff
@@ -8,6 +8,7 @@
 *.sql
 
 # Application
+web/.well-known/security.txt
```

Comment:
Hey, general question, why are we ignoring this file here? As I understand it you are just 301 redirecting to this file in NGINX...does this file ever get generated in our site files thus need to be ignored?

Reply:
The WordPress-base docker image generates a new copy of the file each time there is a build. There was a confusion in the ticket which lead to this behaviour